test/compatibility/cartridge_upgrade_test.lua

```diff
@@ -8,7 +8,7 @@ local g = t.group()
 local helpers = require('test.helper')
 
 local function version(srv)
-    local version = srv:call('require', {'cartridge.VERSION'})
+    local version = srv:call('require', {'cartridge.version'})
```

Comment:
I understand you idea to unify the approach, but I think that there is a lot of people who uses `require('cartridge.VERSION')` now. For example, in metrics tests
https://github.com/tarantool/metrics/blob/e0576c226ea0b9c88d05878bc2c0ea991e8889b5/test/helper.lua#L17
(and I think there could be modules which depends on it in its code). I don't know, why it's not a `require('cartridge').VERSION`, but I'm sure we shouldn't rework the file structure for now.

Reply:
Please, restore the file structure in auth-extention too with new PR

https://github.com/tarantool/cartridge-auth-extension/commit/fc53f7bd60942e8038d0c9141d36c19bfb1d1c8a

Reply:
Restored, also [opened](https://github.com/tarantool/cartridge-auth-extension/pull/10)